web/src/components/EncountersTable/index.tsx

```diff
@@ -1,21 +1,40 @@
 import { Trans } from '@lingui/macro';
-import { Empty } from 'antd';
+import { Empty, TablePaginationConfig } from 'antd';
 import { ColumnsType } from 'antd/lib/table';
+import { useNavigate } from 'react-router-dom';
 
 import { isLoading, isSuccess, RemoteData } from 'aidbox-react/lib/libs/remoteData';
 
 import { SpinIndicator } from '../Spinner';
 import { Table } from '../Table';
+import s from './EncountersTable.module.scss';
 import { EncounterData } from './types';
 
 interface EncountersTableProps {
     columns: ColumnsType<EncounterData>;
     remoteData: RemoteData<EncounterData[]>;
+    handleTableChange: (pagination: TablePaginationConfig) => Promise<void>;
+    pagination: {
+        current: number;
+        pageSize: number;
+        total: number | undefined;
+    };
+    onRowEnabled?: boolean;
 }
 
 export function EncountersTable(props: EncountersTableProps) {
     const encounterDataListRD = props.remoteData;
 
+    const navigate = useNavigate();
+
+    const onRow = (record: any) => {
```

Comment:
Let's discuss this behaviour tomorrow maybe it would be better to have it the same here and on the start Encounters screen with buttons Open and Video Call